modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -146,6 +146,73 @@ a|An integer.
 m|+++600+++
 |===
 
+== Cloud storage integration settings
+
+Configuration settings enable custom cloud storage and host authority endpoints.
```

Comment:
Cloud integration settings allow you to specify custom Azure blob storage endpoints and host authorities, set the project ID for Google Cloud Storage buckets, and define the desired throughput for transfer operations in Amazon S3.